### PR TITLE
.ci/deps.opam.sh: Pin easy-format to v1.2.0

### PR DESCRIPTION
**Pull request by jayvdb**
Infer compilation fails when using easy-format v1.3.0

Fixes https://github.com/coala/coala-bears/issues/2059

### CODE REVIEW

**Comment by jayvdb** [comment]
This passed on my fork https://travis-ci.org/jayvdb/coala-bears/builds/281128565 , which was a complete rebuild.

File: .ci/deps.opam.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   opam init --y
   opam update
   opam pin add --yes --no-action atdgen 1.10.0
+  opam pin add --yes --no-action easy-format 1.2.0
```
Comment (sils) [inline .ci/deps.opam.sh]: a comment about that would be good IMO so it can be removed when infer fixes that and people know what this is about

**Comment by sils** [comment]
marking WIP as per review

File: .ci/deps.opam.sh (hunk 1 above)
Comment (jayvdb) [reply .ci/deps.opam.sh]: infer is at version 0.13.0
They have fixed it.
This is yet another patch to keep alive a horribly old version.

**Comment by Mixih** [comment]
ack 71ed4df

**Pull request merged**